### PR TITLE
Added bzip2 as a dependency of freetype.

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
When looking at the libraries loaded by some programs I found that freetype imported the system bzip2 library. This change fixes this issue for the foss-2016a versions.
I also brought the formatting of the 2.6.2 and 2.6.3 easyconfigs in line with each other, and with the intel-2016a versions.

If this fix is OK it would be a good idea to also update the intel-2016* versions with the bzip2 dependency.

### CODE REVIEW

**Comment by wpoely86** [comment]
It's an optional dep. Freetype will pick it up when it's there. But we can add it, sure.

File: easybuild/easyconfigs/f/freetype/freetype-2.6.2-foss-2016a.eb [hunk 1]
```diff
@@ -11,10 +11,14 @@ toolchain = {'name': 'foss', 'version': '2016a'}
 source_urls = [GNU_SAVANNAH_SOURCE]
 sources = [SOURCE_TAR_GZ]
 
-dependencies = [('libpng', '1.6.21')]
+dependencies = [
+                ('libpng', '1.6.21'),
+                ('bzip2', '1.0.6'),
+               ]
```
Comment (wpoely86) [inline easybuild/easyconfigs/f/freetype/freetype-2.6.2-foss-2016a.eb]: please fix the indenting (we stick to pep8 rules).

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2062.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d2baaa62e6f351990c1199a442679279 for a full test report.

File: easybuild/easyconfigs/f/freetype/freetype-2.6.2-foss-2016a.eb [hunk 2]
```diff
@@ -12,9 +12,9 @@ source_urls = [GNU_SAVANNAH_SOURCE]
 sources = [SOURCE_TAR_GZ]
 
 dependencies = [
-                ('libpng', '1.6.21'),
-                ('bzip2', '1.0.6'),
-               ]
+    ('libpng', '1.6.21'),
+    ('bzip2', '1.0.6'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/f/freetype/freetype-2.6.2-foss-2016a.eb]: And while you're at it, please add `zlib` too :wink:

**Comment by boegel** [comment]
@FokkeDijkstra change made consistent and also added `zlib` as a dep (as suggested by @wpoely86) in https://github.com/rug-cit-ris/easybuild-easyconfigs/pull/2, please check/merge?

File: easybuild/easyconfigs/f/freetype/freetype-2.6.2-foss-2016a.eb (hunk 2 above)
Comment (FokkeDijkstra) [inline easybuild/easyconfigs/f/freetype/freetype-2.6.2-foss-2016a.eb]: zlib is already pulled in by libpng. Is it wise to also add it to freetype?

Comment (boegel) [inline easybuild/easyconfigs/f/freetype/freetype-2.6.2-foss-2016a.eb]: @FokkeDijkstra it doesn't hurt, since it's a direct dependency of freetype

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2062.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ab52922887b2a483b5dddeff180e9a50 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
gligar03.gligar.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5245f799542933f34fea079aa32ac80b for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2485.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/812d3d1738818262f6ae5555daa42c71 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @FokkeDijkstra!

**Pull request merged**